### DISCUSSION TITLE
For Each Data is missing from blocks

### DISCUSSION THREAD

**Discussion by domenicr** [postmanlabs/postman-flows#439] [Defect]
FOR EACH block for data is no longer available on new flows but is available on old flows that have FOR EACH

**Comment by domenicr** [comment]
FOR EACH block for data is no longer available on new flows but is available on old flows that have FOR EACH

**Comment by viko42** [comment]
Use the block "For" instead

**Comment by domenicr** [comment]
For does not recognize my send request response

**Reply by viko42** [reply]
You should show us an example of the flow you want to make
